dataactcore/models/jobModels.py

```diff
@@ -196,8 +196,9 @@ class Comment(Base):
     submission_id = Column(Integer, ForeignKey("submission.submission_id", name="fk_submission", ondelete="CASCADE"),
                            nullable=False)
     submission = relationship(Submission, uselist=False, cascade="delete")
-    file_type_id = Column(Integer, ForeignKey("file_type.file_type_id", name="fk_file_type"), nullable=False)
+    file_type_id = Column(Integer, ForeignKey("file_type.file_type_id", name="fk_file_type"), nullable=True)
     file_type = relationship(FileType, uselist=False)
+    submission_comment = Column(Boolean, nullable=False, default=False, server_default="False")
```

Comment:
I'm curious why we need a boolean flag here actually. We could just search on `file_type_id IS NULL` if we want submission comments.

Reply:
That's definitely doable. The intent was to be explicit/obvious to future developers. If we move to that, I can add an extra comment in the model to account for that.